### DISCUSSION TITLE
Mismatch Results

### DISCUSSION THREAD

**Discussion by xy21hb** [luancarvalhomartins/PyAutoFEP#109] [Q&A]
Dear all,

I rerun the tutorial for a second time with exactly the same parameters, but the results look so different, any idea?

   FXR_12→FXR_85        17.2±0.8 kcal/mol    17.8±0.9 kcal/mol   ΔΔG = -0.6±0.9 kcal/mol
      FXR_12→FXR_84        -2.0±0.6 kcal/mol    -2.7±0.7 kcal/mol   ΔΔG = 0.8±0.6 kcal/mol
      FXR_12→FXR_88        0.7±378.4 kcal/mol   -6.6±1.1 kcal/mol   ΔΔG = 7.3±20.5 kcal/mol
      FXR_12→FXR_76        -5.8±2.7 kcal/mol    -6.1±0.9 kcal/mol   ΔΔG = 0.3±1.6 kcal/mol
      FXR_12→FXR_74         1.0±0.2 kcal/mol     0.9±0.2 kcal/mol   ΔΔG = 0.1±0.2 kcal/mol
================= ΔΔG to FXR_12 ==================
           Molecule               ΔΔG   
            FXR_12             0.0 kcal/mol
            FXR_74             0.1 kcal/mol
            FXR_76             0.3 kcal/mol
            FXR_84             0.8 kcal/mol
            FXR_85             -0.6 kcal/mol
            FXR_88             7.3 kcal/mol

=======================================
  FXR_12→FXR_74         1.0±0.0 kcal/mol     1.0±0.0 kcal/mol   ΔΔG = 0.0±0.0 kcal/mol
      FXR_12→FXR_88        -5.8±0.3 kcal/mol    -5.9±0.3 kcal/mol   ΔΔG = 0.2±0.3 kcal/mol
      FXR_12→FXR_76        -5.7±0.1 kcal/mol    -5.3±0.1 kcal/mol   ΔΔG = -0.3±0.1 kcal/mol
      FXR_12→FXR_84        -3.2±0.1 kcal/mol    -2.7±0.1 kcal/mol   ΔΔG = -0.6±0.1 kcal/mol
      FXR_12→FXR_85        17.3±0.1 kcal/mol    16.7±0.1 kcal/mol   ΔΔG = 0.6±0.1 kcal/mol
================= ΔΔG to FXR_12 ==================
           Molecule               ΔΔG   
            FXR_12             0.0 kcal/mol
            FXR_74             0.0 kcal/mol
            FXR_76             -0.3 kcal/mol
            FXR_84             -0.6 kcal/mol
            FXR_85             0.6 kcal/mol
            FXR_88             0.2 kcal/mol

**Comment by luancarvalhomartins** [comment]
That's expected behavior not related to PyAutoFEP itself, but to the underlying theory of MD sampling. The results will always vary. For some discussion on the statistics of the results, see:
- https://pubs.acs.org/doi/abs/10.1021/acs.jctc.1c00194
- https://pubs.acs.org/doi/10.1021/acs.jcim.0c00900
- https://pubs.acs.org/doi/10.1021/ja512751q
